frontend/src/tabs/canvas/ExtraPixelsPanel.css

```diff
@@ -58,6 +58,8 @@
   justify-content: center;
 
   /* TODO: Fix scrolling */
+  height: 100%;
+  max-height: 240px;
```

Comment:
Were you able to test this?

Reply:
No I wasn't. Does it work now? But it should work

Reply:
No it doesn't

Reply:
You can hard code the users extra pixels and faction pixels data to something in app.js if that helps testing too.